examples/plot_simulate_evoked.py

```diff
@@ -81,13 +127,29 @@
 print('Mean spike rates for individual trials:')
 print(trial_rates)
 
-
 ###############################################################################
 # Now, let us try to make the exogenous driving inputs to the cells
-# synchronous and see what happens
-
-params.update({'sync_evinput': True})
-net_sync = Network(params)
+# synchronous and see what happens. This is achieved by setting sigma=0.
+
+net_sync = Network(params, initialise_hnn_drives=False)
+
+# Distal evoked drive, use same weigths as above
+net_sync.add_evoked_drive(
+    'evdist1', mu=63.53, sigma=0, numspikes=1, weights_ampa=weights_ampa_d1,
+    weights_nmda=weights_nmda_d1, location='distal', seedcore=2,
+    space_constant=3., dispersion_time=dispersion_time_d1)
```

Comment:
don't show `space_constant` in the example since we don't want most users to fiddle with this parameter.

Reply:
Agreed. But on the same token, should we not also remove `seedcore=2`? It was there to reproduce HNN GUI, but it seems out of place now that we no longer even attempt to (quantitatively) match the GUI output?

Reply:
In 169f979:
- removed explicit mention of `seedcore` in example
- changed default to 2